modules/python/test/test_misc.py

```diff
@@ -211,7 +211,7 @@ def test_20968(self):
 
     def test_parse_to_bool_convertible(self):
         try_to_convert = partial(self._try_to_convert, cv.utils.dumpBool)
-        for convertible_true in (True, 1, 64, np.bool(1), np.int8(123), np.int16(11), np.int32(2),
+        for convertible_true in (True, 1, 64, np.int8(123), np.int16(11), np.int32(2),
```

Comment:
Do you intentionally skipped `bool(1)`?

Reply:
I removed bool(1) because it is True, which is already in the tuple.